test/extended/util/framework.go

```diff
@@ -567,7 +567,7 @@ func WaitForAnImageStream(client client.ImageStreamInterface,
 
 // WaitForAnImageStreamTag waits until an image stream with given name has non-empty history for given tag.
 func WaitForAnImageStreamTag(oc *CLI, namespace, name, tag string) error {
-	waitTimeout := time.Second * 60
+	waitTimeout := time.Minute * 10
```

Comment:
we need to be careful when raising timeouts like this. if things go wrong, it can make the entire test run take much much longer.  Do we need to make the timeout an argument instead?